idempotency/src/main/kotlin/com/ritense/idempotency/service/IdempotencyEventService.kt

```diff
@@ -0,0 +1,14 @@
+package com.ritense.idempotency.service
+
+import com.ritense.idempotency.domain.IdempotencyEvent
+import com.ritense.idempotency.repository.IdempotencyEventRepository
+
+class IdempotencyEventService(
+    private val idempotencyEventRepository: IdempotencyEventRepository,
+) {
+    fun check(consumer: String, eventId: String) =
```

Comment:
This module is to ensure at most once processing. So would rename check to isProcessed